Packs/Office365AndAzureAuditLog/Integrations/MicrosoftPolicyAndComplianceAuditLog/README.md

```diff
@@ -21,6 +21,13 @@ Use the integration to get logs from the O365 service.
     3. Run the ***o365-auditlog-auth-test*** command to verify that the authorization process was implemented correctly.
 
 
+## Required Permissions To Search Audit Logs
+- Minimum Required Exchange roles are **Audit Logs** or **View-Only Audit Logs**
+- Go to [The Microsoft Admin Portal](https://admin.microsoft.com/Adminportal#/homepage)
+- Click **Show All** --> **Roles** --> **Roles Assignments** --> **Exchange section**.
+- Click **Add role group** --> Choose for name and description --> Choose Audit Logs or View-Only Audit Logs Roles --> Choose the members --> Add role group
+- For more information --> [How to assign permissions to search the audit log](https://docs.microsoft.com/en-us/microsoft-365/compliance/set-up-basic-audit?view=o365-worldwide#step-2-assign-permissions-to-search-the-audit-log)
```

Comment:
Stay consistent, please - either, all of your bullet points should end with a period or none of them should. Based on the other sections in the README it appears that ending with a period is the way to go.